packages/reactstrap-validation-date/AvDateRange.js

```diff
@@ -205,16 +205,19 @@ export default class AvDateRange extends Component {
     }
   };
 
-  validateDistance = (end, values) => {
-    if (values[this.props.start.name] && end && this.props.distance) {
-      const start = moment(new Date(values[this.props.start.name]));
+  validateDistance = end => {
```

Comment:
Let's unit test this.

Reply:
Since this is currently on snapshot testing, and you mentioned that its due for a refactor/cleanup, I figured unit testing would be a part of that push?

Reply:
Right, eventually we will have to change this component to a 508 Compliant Date Picker or refactor it so that it will be 508 compliant. At least for all the changes we start making in these repos there should be a unit test for each bug fix because if this issue came up then its apparent that we didn't cover all of the code with the current tests.

We are going to push to get away from snapshot testing. `react-testing-library` helps with rendering and getting items from the virtual DOM as well as triggering different events a lot better than way `react-test-renderer` and `enzyme` can do. We should be testing that certain events are happening and the user can activate certain events rather than testing if a component looks like its snapshot since it can change from the different classes or underlying components.